.browserslistrc

```diff
@@ -0,0 +1 @@
+defaults
```

Comment:
Do we need this?

Reply:
I don't know. This is a file that rails auto-generated. I assumed that Rails did this for a reason.

Reply:
Hmm... would you do some research to find the reason? Would be interesting :)

Reply:
> With @babel/preset-env you can define which browsers your app supports. You do that by editing the .browserlistrc. Most tutorials and docs recommend to start with a generic configuration that supports most commonly used browsers:

[https://blog.jakoblind.no/babel-preset-env-custom-browserlistrc/](https://blog.jakoblind.no/babel-preset-env-custom-browserlistrc/)

This seems to be a babel-thing.